.github/stale.yml

```diff
@@ -0,0 +1,62 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 45
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 5
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: []
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - pinned
+  - security
+  - "work in progress"
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: false
+
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: false
+
+# Set to true to ignore issues with an assignee (defaults to false)
+exemptAssignees: false
+
+# Label to use when marking as stale
+staleLabel: stale
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  Hi :smile:, this issue has been automatically marked as stale because it has not had
+  recent activity. Don't worry you can continue to work on this and ask @publiclab/reviewers 
+  to add "work in progress" label :tada: . It will be closed if no further activity occurs :100: . 
```

Comment:
```suggestion
  to add "work in progress" label :tada: . Otherwise, it will be closed if no further activity occurs in 5 days  -- but you can always re-open it if you like! :100:
```

Reply:
How does this look? Sorry @sagarpreet-chadha I missed this! If this looks good, we can go ahead!